patches/webpack-dev-server+3.11.1.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/node_modules/webpack-dev-server/client/utils/createSocketUrl.js b/node_modules/webpack-dev-server/client/utils/createSocketUrl.js
+index f214a2d..506ab14 100644
+--- a/node_modules/webpack-dev-server/client/utils/createSocketUrl.js
++++ b/node_modules/webpack-dev-server/client/utils/createSocketUrl.js
+@@ -65,7 +65,8 @@ function getSocketUrl(urlParts, loc) {
+   // they are not provided
+ 
+ 
+-  var sockHost = query.sockHost || hostname;
++  var sockProtocol = protocol || loc.protocol;
```

Comment:
fixes `Failed to construct WebSocket: invalid url`
newest dev server version parses relative urls incorrectly, omits host+protocol parts
as a result, websocket urls also becomes relative which isn't supported 

i've added fallback to protocol + host from window.location (if they were parsed incorrectly)